server/controllers/stock/index.js

```diff
@@ -143,6 +143,10 @@ async function createStock(req, res, next) {
       transaction.addQuery('CALL PostStockMovement(?)', [postingParams]);
     }
 
+    if (!isExit) {
+      transaction.addQuery('CALL RecomputeStockValue(NULL);');
```

Comment:
I don't think we need to compute this for the stock entry from transfer.

We might need to look up the previous record to give the unit cost associated with that lot for the exit by transfer.